fiftyone/core/cli.py

```
@@ -3632,22 +3652,35 @@ class PluginsInfoCommand(Command):
     @staticmethod
     def execute(parser, args):
-        _print_plugin_info(args.name)
+        _print_plugin_info(args.name_or_dir)
 
 
-def _print_plugin_info(name):
-    plugin_defintion = fop.get_plugin(name)
+def _print_plugin_info(name_or_dir):
+    try:
+        pd = fop.get_plugin(name_or_dir)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)